srcpkgs/booster/files/kernel-hook-postrm

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+#
+# Kernel post-remove hook for booster.
+#
+# Arguments passed to this script: $1 pkgname, $2 version.
+#
+PKGNAME="$1"
+VERSION="$2"
+
+rm -f boot/initramfs-${VERSION}.img
+exit $?
```

Comment:
Would something like this work? We don't want to ignore errors and we are returning the exit code anyway?
```suggestion
exec rm boot/initramfs-${VERSION}.img
```

Reply:
There's no need for `exec` and there's no need for `exit $?` anyway. The shell exits with the return code if the last command it ran.